ui/src/components/common/DismissibleNotifications.tsx

```diff
@@ -48,11 +48,11 @@ const DismissibleNotification: React.FC<DismissibleNotificationProps> = ({
     const senderRole = notification.contractData.senderRole;
     let name;
 
-    useEffect(() => {
-        setTimeout(() => {
-            notificationDismiss();
-        }, 5000);
-    })
+    // useEffect(() => {
+    //     setTimeout(() => {
+    //         notificationDismiss();
+    //     }, 5000);
+    // })
```

Comment:
uncomment

Reply:
Per Eric, we're going to stop the automatic notification dismissals since there's important info like margin calls. I left it in commented in case we want to add it back in later (or use temporarily when testing.)